README.md

```diff
@@ -687,36 +688,53 @@ npx turbo --help
 
 #### Commands
 
-##### `crypto-fund`
+##### `balance`
 
-Fund a wallet with Turbo Credits by submitting a payment transaction for the crypto amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for top up processing.
+Get the balance of a wallet or native address in Turbo Credits.
 
 Command Options:
 
-- `-v, --value <value>` - Amount of tokens in the token type's smallest unit value to fund the wallet with
+- `-a, --address <address>` - Address to get the balance of
 
 e.g:
 
 ```shell
-turbo crypto-fund --value 0.0001 --token kyve --private-key 'b27...45c'
+turbo balance --address 'crypto-wallet-public-native-address'
```

Comment:
How can you just pass a native address without specifying the chain?

Reply:
here we are defaulting to arweave. but I think it would be possible to regex the address to derive the chain type 🤔

Reply:
I feel like we could make that refactor in payment service side to derive token when not provided then plumb that down to here